requirements/test-requirements.txt

```diff
@@ -1,7 +1,6 @@
-## Small test reqs
+## Dependencies required to run tests
 # Required for testing utilities for parsing pip requirements
 pip>=20.1
-scipy
```

Comment:
Scipy is now part of a module dependency and doesn't need explicit install with the changes?

Reply:
yes :)